minicluster/src/main/java/alluxio/master/LocalAlluxioJobCluster.java

```diff
@@ -71,9 +74,12 @@ public void start() throws IOException, ConnectionFailedException {
    * @throws Exception when the operation fails
    */
   public void stop() throws Exception {
-    LOG.info("Stop Alluxio job service");
-    mWorker.stop();
-    mMaster.stop();
+    if (mIsRunning) {
```

Comment:
what happens if you call stop when it is not running?

Reply:
Then it's a no-op as it should be. Or at least it's safe for it to be a no-op when not started.